src/test/java/uk/gov/dhsc/htbhf/page/GuidancePage.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.dhsc.htbhf.page;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.Map;
+
+/**
+ * Page object for the guidance pages of the application
+ */
+public class GuidancePage {
+
+    private static final String APPLY_PAGE = "Apply for Healthy Start";
+
+    //TODO MRS 2019-08-07: Guidance page enum? Look into this when guidance-pages.feature is done
+    private static final Map<String, String> GUIDANCE_PAGES = Map.of(
```

Comment:
I was going to add this as an enum but didn't want to second guess how we'd use it when its used by the main guidance feature. Happy to do so if anyone wants me to tidy this up now though.